pkg/s3wrapper/client.go

```diff
@@ -524,12 +529,14 @@ func (c *S3Client) uploadBootFiles(ctx context.Context, isoObjectName, minimalIs
 	}
 
 	log.Infof("Starting Base ISO download for %s", isoObjectName)
-	baseIsoPath, err := DownloadURLToTemporaryFile(isoURL)
+	baseIsoPath, err := DownloadURLToFile(isoURL, isoObjectName, rhcosImageChecksum)
```

Comment:
This is currently rewritten only for S3, is it worth to do also for filesystem storage type?